app/scripts/modules/core/src/pipeline/config/templates/TemplatePlanErrors.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import autoBindMethods from 'class-autobind-decorator';
+
+import { IPipelineTemplatePlanError } from 'core/pipeline/config/templates/pipelineTemplate.service';
+
+export interface ITemplatePlanErrorsProps {
+  errors: IPipelineTemplatePlanError[];
+}
+
+export interface ITemplatePlanErrorsState { }
+
+@autoBindMethods
+export class TemplatePlanErrors extends React.Component<ITemplatePlanErrorsProps, ITemplatePlanErrorsState> {
```

Comment:
if there is no state, you can just extend `React.Component<ITemplatePlanErrorsProps, void>`